### ISSUE TITLE
(python) Version 4.9.0 tar no longer available

### ISSUE THREAD

**Issue by Raggey** [pulumi/pulumi-random#270]
### What happened?

Currently trying to install v4.9.0 via pypi (python) of pulumi-random via an automated pipeline, and running into a 403 trying to pull the tar

error: [resource plugin random-4.9.0] downloading from : 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-random-v4.9.0-linux-amd64.tar.gz

### Expected Behavior

Expected tar to exist, tar exists for most recent version 4.11.2

### Steps to reproduce

HTTP 403 on the following link: https://get.pulumi.com/releases/plugins/pulumi-resource-random-v4.9.0-linux-amd64.tar.gz 

### Output of `pulumi about`

CLI          
Version      3.34.1
Go Version   go1.17.11
Go Compiler  gc

Host     
OS       ubuntu
Version  22.04
Arch     x86_64

### Additional context

4.9.0 still available on pypi, so bit confused by this

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by AaronFriel** [comment]
Hey @Raggey, thanks for reporting this and my apologies. We had to yank this package because it was causing upgrades to fail (#258). If you're on this version, you may need to do edit the stack manually.

```sh
cd DIR_WITH_PROGRAM
# select the stack you need to edit
pulumi stack select
# export the stack to a file
pulumi stack export --file stack.json
```

Open `stack.json` and find these lines:

```json
                "type": "pulumi:providers:random",
                "inputs": {
                    "version": "4.9.0"
                },
                "outputs": {
                    "version": "4.9.0"
                }
```

Edit both lines with `"version": "4.9.0"` to a greater version that's available, such as 4.11.2.

Then:

```
pulumi stack import --file stack.json
```

**Comment by Raggey** [comment]
Hey @AaronFriel thanks for the quick response!

Didn't realise this had been raised in another ticket. Has that entire version/release been yanked? I'm still seeing in on pypi, so pip will continue to resolve and install this version.

I'll be sure to upgrade to resolve this. Thanks!
